src/lib.rs

```diff
@@ -473,20 +476,55 @@ impl Account {
     ) -> Result<(Account, AccountCredentials), Error> {
         Self::create_inner(
             account,
+            Key::generate()?,
             external_account,
             Client::new(server_url, http).await?,
             server_url,
         )
         .await
     }
 
+    /// Load a new account by private key, with a default or custom HTTP client
+    ///
+    /// https://www.rfc-editor.org/rfc/rfc8555#section-7.3.1
+    ///
+    /// The returned [`AccountCredentials`] can be serialized and stored for later use.
+    /// Use [`Account::from_credentials()`] to restore the account from the credentials.
+    pub async fn load(
+        private_key_pkcs8_der: &[u8],
+        server_url: &str,
+        http: Option<Box<dyn HttpClient>>,
```

Comment:
Not sure the `Option` type really makes sense for `http`? If it is `Some` you don't want to pull in the `DefaultClient` dependendencies; if it is `None` you need to. Which one do you want to use?

Reply:
Ahh, I did not see that was behind a feature gate.